internal/encrypt/init.go

```diff
@@ -0,0 +1,123 @@
+package encrypt
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/inconshreveable/log15"
+
+	"github.com/sourcegraph/sourcegraph/internal/conf"
+)
+
+var cryptObject Encryptor
```

Comment:
This sounds more reasonable to me... but up to you!
```suggestion
var defaultEncryptor Encryptor
```